content/reference/endpoints/clients.md

```diff
@@ -3,6 +3,10 @@
 1. TOC
 {:toc}
 
+## Overview
+
+This resource represents the Clients which effectively are the actual travelers that are related to [Bookings](/reference/endpoints/bookings/). 
```

Comment:
It's not actually the person who confirmed the booking, this might be a different role (e.g. someone paying for the booking but not travelling). This is literally the traveller or maybe even a `Guest` would be a better word

Reply:
yes, guest sounds fine 👍